sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1050,6 +1050,9 @@ def check_sparse_input(EstimatorClass, X, X_sparse, y):
     assert_array_almost_equal(sparse.feature_importances_,
                               auto.feature_importances_)
 
+    assert_array_equal(sparse.predict(X_sparse), dense.predict(X))
+    assert_array_equal(dense.predict(X_sparse), sparse.predict(X))
```

Comment:
Please use `assert_array_almost_equal` to compare arrays of potentially floating point values (when the estimator is a regressor).